flw/src/main/java/org/motechproject/nms/flw/web/FrontLineWorkerImportController.java

```diff
@@ -63,6 +64,12 @@ public void updateFrontLineWorkersLanguage(@RequestParam MultipartFile csvFile)
     @RequestMapping(value = "/update/msisdn", method = RequestMethod.POST)
     @ResponseStatus(HttpStatus.OK)
     public void updateFrontLineWorkersMSISDN(@RequestParam MultipartFile csvFile) {
+
+        // Update FrontLineWorkers by Msisdn import from csv is not supported
+        if(true) {
```

Comment:
I want to merge this thing later in my spare time, which can be deployed along with next deployment.That's why we decided not to remove this thing.That won't  affect our regular deadlines.